### PR TITLE
Enable spotbugs check in module package management service

### PR DESCRIPTION
**Pull request by zymap**
---

*Motivation*

Enable spotbugs checks in the module package management service.

### CODE REVIEW

File: pulsar-package-management/core/pom.xml [hunk 1]
```diff
@@ -48,4 +48,16 @@
...
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <configuration>
+                    <excludeFilterFile>${basedir}/src/main/resources/findbugsExclude.xml</excludeFilterFile>
```
Comment (codelipenghui) [inline pulsar-package-management/core/pom.xml:58]:
```suggestion
                    <excludeFilterFile>${basedir}/src/main/resources/findbugs-exclude.xml</excludeFilterFile>
```

Comment (zymap) [reply pulsar-package-management/core/pom.xml:58]: We are using this file for the exclude filter: https://github.com/apache/pulsar/pull/8784/files#diff-2ece31f925717628e3a3b606d38970a684b39b9e37f15d90fde5cea381420b02

**Review by codelipenghui — APPROVED** [review]

**Review by sijie — APPROVED** [review]

**Pull request merged**